assets/src/ansible/06_run_seqerakit.yml.tpl

```diff
@@ -35,79 +37,19 @@
       export SEQERAKIT_AWS_ACCESS_KEY_ID=$(aws ssm get-parameters --name "/seqera/${app_name}/seqerakit/aws-user" --with-decryption --query "Parameters[*].{Value:Value}" --output text)
       export SEQERAKIT_AWS_SECRET_ACCESS_KEY=$(aws ssm get-parameters --name "/seqera/${app_name}/seqerakit/aws-password" --with-decryption --query "Parameters[*].{Value:Value}" --output text)
       export SEQERAKIT_AWS_ASSUME_ROLE_ARN=$(aws ssm get-parameters --name "/seqera/${app_name}/seqerakit/aws-role" --with-decryption --query "Parameters[*].{Value:Value}" --output text)
+      export TOWER_CODECOMMIT_USER=$(aws ssm get-parameters --name "/seqera/${app_name}/seqerakit/codecommit-user" --with-decryption --query "Parameters[*].{Value:Value}" --output text)
+      export TOWER_CODECOMMIT_PASSWORD=$(aws ssm get-parameters --name "/seqera/${app_name}/seqerakit/codecommit-password" --with-decryption --query "Parameters[*].{Value:Value}" --output text)
+      export TOWER_CODECOMMIT_REGION=$(aws ssm get-parameters --name "/seqera/${app_name}/seqerakit/codecommit-region" --with-decryption --query "Parameters[*].{Value:Value}" --output text)
```

Comment:
As per PR comment, I think this should be changed to `$TOWER_CODECOMMIT_BASEURL`.